packages/client/src/types.ts

```diff
@@ -25,6 +28,7 @@ export interface VercelClientOptions {
   defaultName?: string;
   isDirectory?: boolean;
   skipAutoDetectionConfirmation?: boolean;
+  archive?: ArchiveFormat | undefined;
```

Comment:
I think this can be `archive?: ArchiveFormat` since undefined is implied by the `?`